.github/workflows/lint.yml

```diff
@@ -19,12 +19,9 @@ jobs:
 
       - name: ruff
         run: |
-          ruff --version
```

Comment:
Printing version doesn't seem necessary as the install stage would show the version:
<img width="1424" alt="image" src="https://github.com/user-attachments/assets/33d3dcb3-d54f-46ce-aeb1-37b664ab1263">